gist/gist.py

```diff
@@ -138,19 +138,6 @@ def __init__(self, token, editor=None):
         """
         self.token = token
         self.editor = editor
-        self.proxies = {}
```

Comment:
The reason that that code was added was because it did not pull the information from the environment. That may have changed. However, the documentation I pointed you to suggested that proxies need to be manually passed in when manually creating the Session object. Perhaps I am missing something, but if I create those environmental variables, and create a Session object, it will not have any proxies associated with it.

Regarding unit tests, you are proposing a change to this code base that could cause a regression. You need to demonstrate that you are not introducing a regression.